Code/Editor/Util/ModalWindowDismisser.cpp

```diff
@@ -53,12 +53,12 @@ bool ModalWindowDismisser::eventFilter(QObject* object, QEvent* event)
                 {
                     m_windows.push_back(dialog);
                 }
-                if (!m_dissmiss)
+                if (!m_dismiss)
                 {
                     // Closing the window at the same moment is opened leads to crashes and is unstable,
-                    // so do it after a long 1 ms
-                    QTimer::singleShot(1, this, &ModalWindowDismisser::DismissWindows);
-                    m_dissmiss = true;
+                    // so do it after a long 2 ms (needed for Jenkins to realise the dialog is there in some circumstances)
```

Comment:
uber-nit: we use US English so this would be `realize`

Reply:
Removed